src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -196,14 +217,29 @@ private function getOutputCallback()
     {
         if ($this->getContainer()->get(LocalMachineHelper::class)->useTty() === false) {
             $output = $this->output();
-            return function ($type, $buffer) use ($output) {
-                $output->write($buffer);
+            $stderr = $this->stdErr();
+
+            return function ($type, $buffer) use ($output, $stderr) {
+                if (Process::ERR === $type) {
+                    $stderr->write($buffer);
+                } else {
+                    $output->write($buffer);
+                }
             };
         }
         return function ($type, $buffer) {
         };
     }
 
+    private function stdErr()
```

Comment:
I'm surprised `Robo/IO` doesn't have a shortcut for this.

Reply:
Yeah I should push it up to the upstream.